cut-n-paste/toolbar-editor/egg-toolbars-model.c

```diff
@@ -721,6 +732,8 @@ egg_toolbars_model_load_names_from_resource (EggToolbarsModel *model,
     g_error ("Failed to load XML data from resource %s", path);
 
   root = xmlDocGetRootElement (doc);
+  if (root == NULL)
+    g_error("Failed to get root element");
```

Comment:
code formatting: blank space is missing, before opening parenthesis. g_error_(...